src/main/java/net/william278/velocitab/commands/VelocitabCommand.java

```diff
@@ -0,0 +1,67 @@
+package net.william278.velocitab.commands;
+
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.command.SimpleCommand;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextColor;
+import net.william278.desertwell.AboutMenu;
+import net.william278.desertwell.Version;
+import net.william278.velocitab.Velocitab;
+
+import java.util.List;
+
+public class VelocitabCommand implements SimpleCommand {
+    private final AboutMenu aboutMenu;
+    private final Velocitab plugin;
+
+    public VelocitabCommand(Velocitab plugin) {
+        this.plugin = plugin;
+        aboutMenu = AboutMenu.create("Velocitab")
+                .withDescription("Velocitab is a super-simple Velocity TAB menu plugin that uses scoreboard team client-bound packets to actually sort player lists without the need for a backend plugin.")
```

Comment:
```suggestion
                .withDescription("A super-simple (sorted!) Velocity TAB menu plugin")
```
I prefer to use the short description (in this case, from the `velocity-plugin.json`) file, here.

It'd probably be better if we actually fetched this from the PluginContainer, though, so it can be edited in just one place (in said file). We can get it from within the plugin main class using:
```java
pluginContainer.getDescription().getDescription().ifPresent(descriptionString -> DESCRIPTION = descriptionString);
```
And then we could use it here.